packages/vike-vue/src/components/Head/Head-server.ts

```diff
@@ -0,0 +1,26 @@
+export { Head }
+
+// Same as ./Head-client.ts but importing useConfig-server.js
+
+import { useConfig } from '../../hooks/useConfig/useConfig-server.js'
+import { defineComponent } from 'vue'
+
+/**
+ * Add arbitrary `<head>` tags.
+ *
+ * (The children are teleported to `<head>`.)
+ *
+ * https://vike.dev/Head
+ */
+const Head = defineComponent({
+  name: 'Head',
+  inheritAttrs: false,
+  setup:
+    (_props, { slots }) =>
+    () => {
```

Comment:
This is executed in the render function - I think it would be better to use watchers
If Head is rendered inside a component that uses scoped styles the rendered tags will have the scope markers `data-v-...`

Reply:
I don’t think we need watchers, since this only runs on the server side.

> If Head is rendered inside a component that uses scoped styles the rendered tags will have the scope markers `data-v-...`

Thanks, I’ve updated the code similarly to yours.